src/index.html

```diff
@@ -1,13 +1,20 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Antisnake</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Antisnake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Antisnake</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="container">
+      <div class="block block__styles--1">1</div>
+      <div class="block block__styles--2">2</div>
+      <div class="block block__styles--3">3</div>
```

Comment:
```suggestion
      <div class="block">1</div>
      <div class="block">2</div>
      <div class="block">3</div>
```
let's not make separate classes for each block